packages/longhorn/longhorn.patch

```diff
@@ -0,0 +1,374 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/Chart.yaml packages/longhorn/charts/Chart.yaml
+--- packages/longhorn/charts-original/Chart.yaml
++++ packages/longhorn/charts/Chart.yaml
+@@ -4,24 +4,29 @@
+ home: https://github.com/longhorn/longhorn
+ icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.svg?sanitize=true
+ keywords:
+-- longhorn
+-- storage
+-- distributed
+-- block
+-- device
+-- iscsi
++  - longhorn
++  - storage
++  - distributed
++  - block
++  - device
++  - iscsi
+ kubeVersion: '>=v1.14.0-r0'
+ maintainers:
+-- email: maintainers@longhorn.io
+-  name: Longhorn maintainers
+-- email: sheng@yasker.org
+-  name: Sheng Yang
++  - email: maintainers@longhorn.io
++    name: Longhorn maintainers
++  - email: sheng@yasker.org
++    name: Sheng Yang
+ name: longhorn
+ sources:
+-- https://github.com/longhorn/longhorn
+-- https://github.com/longhorn/longhorn-engine
+-- https://github.com/longhorn/longhorn-instance-manager
+-- https://github.com/longhorn/longhorn-manager
+-- https://github.com/longhorn/longhorn-ui
+-- https://github.com/longhorn/longhorn-tests
++  - https://github.com/longhorn/longhorn
++  - https://github.com/longhorn/longhorn-engine
++  - https://github.com/longhorn/longhorn-instance-manager
++  - https://github.com/longhorn/longhorn-manager
++  - https://github.com/longhorn/longhorn-ui
++  - https://github.com/longhorn/longhorn-tests
+ version: 1.0.1
++annotations:
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/namespace: longhorn-system
++  catalog.cattle.io/release-name: longhorn
++  catalog.cattle.io/ui-component: longhorn
+diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/crds/crds.yaml packages/longhorn/charts/crds/crds.yaml
+--- packages/longhorn/charts-original/crds/crds.yaml
++++ packages/longhorn/charts/crds/crds.yaml
```

Comment:
The `prepare-crd` script needs currently does not expect multiple CRDs to be located within the same file due to the way we collect the CRDs to be added to the validation script: https://github.com/rancher/charts/blob/531420dcf5ed3089d3dbca6f312304cf853b9b6d/scripts/prepare-crds#L48-L54

We should split each CRD into its own file for now. 

Created an issue to track the change required to support this: https://github.com/rancher/rancher/issues/28320

Reply:
I am still unclear on why we need to include the CRD separately. Pinged you separately for a few questions.